subprojects/packagefiles/x-plane-sdk/meson.build

```diff
@@ -0,0 +1,128 @@
+project(
+    'x-plane-sdk',
+    'c',
+    version: '4.0.1',
+    license: 'MIT',
+    default_options: 'warning_level=0',
```

Comment:
Added in 0.50.0: https://github.com/mesonbuild/meson/commit/a9e63568fe4a1ef420cdb8bacb85fbf6d621dc0d

Reply:
One day I'll get the version number right.